dropbox-sdk-android/src/main/java/com/dropbox/core/android/internal/DropboxAuthIntent.kt

```diff
@@ -32,7 +31,14 @@ internal object DropboxAuthIntent {
             putExtra(EXTRA_SESSION_ID, mState.mSessionId)
             putExtra(EXTRA_CALLING_PACKAGE, packageName)
             putExtra(EXTRA_AUTH_STATE, stateNonce)
+
             mState.mTokenAccessType?.apply {
+                val queryParams = QueryParamsUtil.createExtraQueryParams(
+                    tokenAccessType = mState.mTokenAccessType,
+                    scope = mState.mScope,
+                    includeGrantedScopes = mState.mIncludeGrantedScopes,
+                    pkceManagerCodeChallenge = mState.mPKCEManager.codeChallenge
+                )
```

Comment:
This is the actual fix.  The original java code make sure `mState.mTokenAccessType` was non-null.